mlflow/models/cli.py

```diff
@@ -33,8 +33,9 @@ def commands():
 @cli_args.INSTALL_MLFLOW
 def serve(model_uri, port, host, workers, no_conda=False, install_mlflow=False):
     """
-    Serve a model saved with MLflow by launching a webserver on the specified host and port. For
-    information about the input data formats accepted by the webserver, see the following
+    Serve a model saved with MLflow by launching a webserver on the specified host and port. 
+    The command supports models with the ``python_function`` or ``crate`` (R Function) flavor.
```

Comment:
@dmatrix - what do you think? this will show up in the docs once those are updated (https://www.mlflow.org/docs/latest/cli.html#mlflow-models-serve)

Reply:
Agreed. The extra line stipulates what flavors we support.